test/files/rf007/README.MD

```diff
@@ -0,0 +1,10 @@
+|-----------------------------------------------------------------------------------|-----------------|-----------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| pass-ri007-stationing-property                                                    | success         | n.a.                                                                                                      |                                                                                                    |
+| fail-ri007-scenario01-no-name-attribute-of-ifcreferent-default                    | fail            | On instance #532=IfcReferent('13nJTlK...ATION.) the value doesn't exist                                   | A value was expected for Name attribute of #532=IfcReferent(), but $ was found                     |
+| fail-ri007-scenario01-no-name-attribute-of-ifcreferent-empty                      | fail            | On instance #532=IfcReferent('13nJTlK...ION.,$) the value doesn't exist                                   | A value was expected for Name attribute of #532=IfcReferent(), but '' was found                    |
+| fail-ri007-scenario02-no-name-attribute-of-ifcpropertyset-default                 | fail            | On instance (#536=IfcPropertySet('0oZ...537)),) the value doesn't exist                                   | A value was expected for Name attribute of #536=IfcPropertySet(), but $ was found                  |
+| fail-ri007-scenario02-no-name-attribute-of-ifcpropertyset-empty                   | fail            | On instance (#536=IfcPropertySet('0oZ...537)),) the value doesn't exist                                   | A value was expected for Name attribute of #536=IfcPropertySet(), but '' was found                 |
+| fail-ri007-scenario03-wrong-name-value                                            | fail            | On instance #537=IfcPropertySingleVal...3.1),$) the following invalid value for Name has been found: Body | The expected RepresentationType value was Curve3D.                                                 |
+| fail-ri007-scenario04-no-nominalvalue-attribute-of-ifcpropertysinglevalue-default | fail            | On instance ((#537=IfcPropertySingleV...,$),),) the value doesn't exist                                   | A value was expected for NominalValue attribute of #537=IFCPROPERTYSINGLEVALUE(), but $ was found  |
+| fail-ri007-scenario04-no-nominalvalue-attribute-of-ifcpropertysinglevalue-empty   | fail            | On instance ((#537=IfcPropertySingleV...,$),),) the value doesn't exist                                   | A value was expected for NominalValue attribute of #537=IFCPROPERTYSINGLEVALUE(), but '' was found |
```

Comment:
What happens if the property set "Pset_Stationing" contains more than one property?